modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -415,15 +412,13 @@ private CompletableFuture<Void> pushAuthenticationConfigToCluster(CmgRaftService
                     if (state == null) {
                         LOG.info("No CMG state found in the Raft storage");
                         return completedFuture(null);
-                    } else if (state.restAuthToApply() == null) {
-                        // auth config has already been successfully pushed to the distributed configuration
-                        LOG.info("No REST auth configuration found in the Raft storage");
+                    } else if (state.clusterConfigurationToApply() == null) {
```

Comment:
I think that the method `pushAuthenticationConfigToCluster` deserves renaming too